static/js/models/thing-model.js

```diff
@@ -127,39 +127,41 @@ class ThingModel extends Model {
   /**
    * Initialize websocket.
    */
-  initWebsocket() {
-    if (this.closingWs) {
-      return;
-    }
-
+  initWebSocket(globalWs) {
     if (!this.hasOwnProperty('href')) {
       return;
     }
 
-    const wsHref = this.href.href.replace(/^http/, 'ws');
-    this.ws = new WebSocket(`${wsHref}?jwt=${API.jwt}`);
+    if (globalWs) {
+      this.ws = globalWs;
+      this.personalWebSocket = false;
+    } else {
+      const wsHref = this.href.href.replace(/^http/, 'ws');
```

Comment:
I don't think this non-global logic is ever used.

Reply:
I thought it would be nice to leave in case the gateway ever connects directly to things but I'll just tear it out and simplify

Reply:
Although now closePermanently is never called